<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagesWidget</class>
 <widget class="QWidget" name="ImagesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="MainLayout">
   <item>
    <widget class="QGraphicsView" name="OriginalImg"/>
   </item>
   <item>
    <widget class="QGraphicsView" name="ProcessedImg"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
